techniques/system/common/1.0/failsafe.st

```diff
@@ -126,6 +126,12 @@ bundle common g
 
 # definition of the node roles
   &NODEROLE&
+
+  classes:
+      # OS classes for compatibility
```

Comment:
shouldn't there be a scope => namespace ?

Reply:
Nope, it is a bundle common where classes are global by default.